plugins/SliceInfoPlugin/MoreInfoWindow.qml

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// PluginBrowser is released under the terms of the LGPLv3 or higher.
```

Comment:
That may be true, but what is Cura released under?

Reply:
Fixed in 50153b43a891e1a8817df1c7d87646291b4f18b4